### PR TITLE
[Shop][UI] Fix styles path for the shop

### PR DESCRIPTION
**Pull request by pjedrzejewski**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Related tickets | fixes #X, partially #Y, mentioned in #Z
| License         | MIT

### CODE REVIEW

File: src/Sylius/Bundle/ShopBundle/Gulpfile.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ var paths = {
         ],
         sass: [
             '../UiBundle/Resources/private/sass/**',
```
Comment (pamil) [inline src/Sylius/Bundle/ShopBundle/Gulpfile.js]: Will it work? With Sylius installed as single components (not `sylius/sylius` replacing all the other packages), the UiBundle will be at `vendor/sylius/ui-bundle/` and ShopBundle at `vendor/sylius/shop-bundle/`.

File: src/Sylius/Bundle/ShopBundle/Gulpfile.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ var paths = {
...
-            'ShopBundle/Resources/private/scss/**'
+            'Resources/private/scss/**'
```
Comment (pamil) [inline src/Sylius/Bundle/ShopBundle/Gulpfile.js]: Side note: we have `sass` folder in UiBundle and `scss` here in ShopBundle.

Comment (pjedrzejewski) [inline src/Sylius/Bundle/ShopBundle/Gulpfile.js]: Yeah, I will clean this up when finalizing the UI.

File: src/Sylius/Bundle/ShopBundle/Gulpfile.js (hunk 1 above)
Comment (pjedrzejewski) [inline src/Sylius/Bundle/ShopBundle/Gulpfile.js]: If you use Shop/Admin/Api, which all depend on Core, I think we won't recommend using them as standalone components and maybe we even remove their composer.json and recommend depending on ``sylius/sylius``, which solves this and many other problems. Thoughts?

**Comment by michalmarcinkowski** [comment]
👍

**Pull request merged**